todo_app/templates/index.html

```diff
@@ -11,8 +11,8 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        {% for item in items%}
-          <li>{{item.id}} {{item.name}}: {{item.status}}</li>
+        {% for item in view_model.items %}
```

Comment:
Worth noting that the exercise is suggesting we switch over to split this into the different properties, e.g. `{% for item in view_model.to_do_items %}` - fine not to do this as long as we acknowledge that our ViewModel & Tests are code that has been written purely for the sake of learning & testing, and might as well all be deleted! 😄